modules/exploits/linux/http/pulse_secure_cmd_exec.rb

```diff
@@ -1,180 +0,0 @@
-##
```

Comment:
Why did you delete this module?

Reply:
your guess is as good as mine.

Reply:
Ah, hell, I just saw https://github.com/rapid7/metasploit-framework/pull/14368#issuecomment-723722461. Nevermind.